puzzle.js

```diff
@@ -629,6 +665,52 @@ function PuzzleEntry(p) {
         tr.appendChild(td);
     }
 
+    // Copyjack support.
+    //
+    // Reads from inputTd, a td that's part of an interactive puzzle element.
+    // Expects that inputTd.dataset.copyjack is copyTd, a td that's part of this.copyjackVersion.
+    // Styles copyTd such that clipboard copies it correctly.
+    this.processTdForCopyjack = function(inputTd) {
+        const [i, j] = inputTd.dataset.coord.split(",")
```

Comment:
If the page does NOT have any copy-to-clipboard functionality, this should return immediately.